release/platform/platform.go

```diff
@@ -536,7 +536,7 @@ var platforms = []Platform{
 		OS:        OSLinux,
 		Pkg:       PkgDeb,
 		Repos:     []Repo{RepoEnterprise, RepoOrg},
-		BuildTags: []string{"failpoints"},
+		BuildTags: defaultBuildTags,
```

Comment:
This works as is, I don't think there's any reason we need to not support Kerberos on ubuntu arm variants.

Reply:
I'm pretty sure this wasn't done intentionally when these platforms were added. We probably should've caught it in PR review. Good to fix it now!